tests/preferences/test_preferences_dialog.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from hamster_gtk.preferences import PreferencesDialog
+
+
+class TestPreferencesDialog(object):
+    """Unittests for PreferencesDialog."""
+
+    def test_init(self, dummy_window, app, config):
+        """Make instantiation works as expected."""
+        result = PreferencesDialog(dummy_window, app, config)
+        grid = result.get_content_area().get_children()[0]
+        # This assumes 2 children per config entry (label and widget).
+        assert len(grid.get_children()) / 2 == len(config.keys())
+
```

Comment:
It is actually covered with `get_config`, because `_set_config` is used by the constructor. I added [a simple test](https://github.com/jtojnar/hamster-gtk/commit/99e7b55ed0b299f836361c4b7820d08790624446) but it cannot test anything until I get my hands on `another_config_parametrized`. Which I have no idea how to do.